patches/socket.io-react-hook.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist/chunk-AKZXX5FL.mjs b/dist/chunk-AKZXX5FL.mjs
+index d62c0c43e7e25047e2fea77a11ecfa6d9a1b1df0..c32cbc7500637868a8afbd980d0b75b00209b9c4 100644
+--- a/dist/chunk-AKZXX5FL.mjs
++++ b/dist/chunk-AKZXX5FL.mjs
+@@ -78,7 +78,7 @@ function useSocketEvent(socket, event, options) {
+         cleanup();
+       }
+     };
+-  }, [socket]);
++  }, [socket, onMessage]);
+   return { ...state.current, sendMessage };
+ }
+ var useSocketEvent_default = useSocketEvent;
```

Comment:
How is this fixed in this PR? I don't see it on Dockerfile

Reply:
apparently pulumi building the image was causing alex's issues